package.json

```diff
@@ -21,23 +21,24 @@
   },
   "license": "MIT",
   "dependencies": {
-    "async": "~0.9.0",
-    "commondir": "0.0.1",
-    "debug": "^2.0.0",
-    "lodash": "^3.6.0",
-    "semver": "^4.1.0",
-    "toposort": "^0.2.10"
+    "async": "^0.9.2",
+    "bluebird": "^3.3.5",
+    "commondir": "1.0.1",
+    "debug": "^2.2.0",
+    "lodash": "^4.11.1",
+    "semver": "^5.1.0",
+    "toposort": "^0.2.12"
   },
   "devDependencies": {
-    "browserify": "^4.1.8",
-    "chai": "^1.10.0",
-    "coffee-script": "^1.8.0",
-    "coffeeify": "^0.7.0",
+    "browserify": "^4.2.3",
     "eslint": "^2.5.3",
     "eslint-config-loopback": "^1.0.0",
-    "fs-extra": "^0.12.0",
-    "loopback": "^2.16.3",
-    "mocha": "^1.19.0",
-    "supertest": "^0.14.0"
+    "chai": "^3.5.0",
```

Comment:
Please keep the dev-dependencies sorted. (`browserify`, `chai`, `coffee*`, `eslint*`, ...)